### PR TITLE
ORC-727: Update `Java Tools` documentation

### PR DESCRIPTION
**Pull request by guiyanakuang**
### What changes were proposed in this pull request?

Update `Java Tools` documentation.
Added description of count / key / version.
Commands and parameters are listed in order.
Fix the problem that the page does not display parameters.

### Why are the changes needed?

To make it easier for users to use java-tool.

### How was this patch tested?

No need to test.

### CODE REVIEW

**Review by dongjoon-hyun — COMMENTED** [review]
Wow, thank you so much, @guiyanakuang !

File: site/_docs/java-tools.md [hunk 1]
```diff
@@ -11,43 +11,136 @@ supports both the local file system and HDFS.
...
 % java -jar orc-tools-X.Y.Z-uber.jar <sub-command> <args>
 ~~~
 
+## Java Convert
+
+The convert command reads several JSON files and converts them into a
+single ORC file.
+
+-e,--escape `<escape>`
+  : Sets CSV escape character
```
Comment (dongjoon-hyun) [inline site/_docs/java-tools.md:35]: In HTML, `--escape` becomes `-escape`. Shall we use a different way to express this?
Screen Shot 2021-08-31 at 8 18 59 AM

File: site/_docs/java-tools.md [hunk 2]
```diff
@@ -11,43 +11,136 @@ supports both the local file system and HDFS.
...
+
+-t,--timestampformat `<timestampformat>`
+  : Sets timestamp Format
+  
+The automatic JSON schema discovery is equivalent to the json-schema tool
+below.
+
+## Java Count
+
+The count command recursively find *.orc and print the number of rows. The parameter value can be a space separated file paths string.
```
Comment (dongjoon-hyun) [inline site/_docs/java-tools.md:69]: Shall we remove the following? Actually, it's not a single parameter. When it's separated by space, we call them `parameters`.
> `The parameter value can be a space separated file paths string.`

File: site/_docs/java-tools.md [hunk 3]
```diff
@@ -11,43 +11,136 @@ supports both the local file system and HDFS.
...
+
+## Java Data
+
+The data command prints the data in an ORC file as a JSON document. Each
+record is printed as a JSON object on a line. Each record is annotated with
+the fieldnames and a JSON representation that depends on the field's type.
+
+-h,--help
```
Comment (dongjoon-hyun) [inline site/_docs/java-tools.md:77]: ditto. `--` becomes `-` in HTML generation.

File: site/_docs/java-tools.md [hunk 4]
```diff
@@ -201,47 +294,21 @@ Padding ratio: 0%
...
+  : Print exceptions
 
--t
-  : Print the schema as a Hive table declaration
+## Java Version
 
--h
-  : Print help
\ No newline at end of file
+The version command print the version of this ORC tool.
```
Comment (dongjoon-hyun) [inline site/_docs/java-tools.md:314]: `print` -> `prints`

File: site/_docs/java-tools.md (hunk 1 above)
Comment (guiyanakuang) [reply site/_docs/java-tools.md:35]: Use ``might work, I'll test it later

Comment (dongjoon-hyun) [reply site/_docs/java-tools.md:35]: Thanks!

Comment (guiyanakuang) [reply site/_docs/java-tools.md:35]: I've updated java-tools.md. My local preview html confirms that it's fixed.

File: site/_docs/java-tools.md (hunk 2 above)
Comment (guiyanakuang) [reply site/_docs/java-tools.md]: I removed this phrase. BTW way count command does not give any error if there is no path. Also I would like to support specifying a single file, not necessarily a directory.

File: site/_docs/java-tools.md (hunk 3 above)
Comment (guiyanakuang) [reply site/_docs/java-tools.md]: Fix it in da9740a

File: site/_docs/java-tools.md (hunk 4 above)
Comment (guiyanakuang) [reply site/_docs/java-tools.md]: Fix it in da9740a

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Thank you, @guiyanakuang .
Merged to main/1.7.

**Pull request merged**